apps/condo/domains/common/utils/codegeneration/generate.server.utils.js

```diff
@@ -99,11 +104,38 @@ function generateServerUtils (gql) {
         })
     }
 
+    /**
+     * Returns result with flat related IDs, relations would not be resolved because of getById
+     * @param context
+     * @param attrs
+     * @param existingItem
+     * @returns {Promise<*|null|undefined>}
+     */
+    async function createOrUpdateExistingItem (context, attrs, existingItem) {
```

Comment:
I've expected something like: https://docs.djangoproject.com/en/4.0/ref/models/querysets/#get-or-create for this function.

Reply:
got rid of it